### PR TITLE
added new effect to contribution section

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
https://nolik2000.github.io/gci16.fossasia.org/

**Comment by nolik2000** [comment]
Why it always happened???

**Comment by nolik2000** [comment]
What I do incorrectly?

**Comment by ghost** [comment]
I always try to fix it but the result is 0

**Comment by gaeun** [comment]
@nolik2000 This is a cool feature, but it seems like the feature overlaps with the student and contributors sections. Could you fix that? Awesome job though :D

**Comment by animeshsinghweb** [comment]
@nolik2000 There are some merge conflicts, can you fix it up?

File: javascripts/contributors.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      "Prabhdeep29",
+      "Hpdang",
+      "LeeCareGene",
+      "Abhijeetmanohar",
+      "Baidya99",
+      "Nolik2000",
+      "Pipix51",
+      "Codethejason",
+      "Shubham-padia",
+      "Hesapadim",
```
Comment (abishekvashok) [inline javascripts/contributors.js:28]: Don't use static arrays use nithin api bro :)

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Please fix as stated

**Comment by niccokunzmann** [comment]
@nolik2000 Do you need help with this?

**Comment by ghost** [comment]
@niccokunzmann  yeah I do not clearly know how to do it

**Comment by ghost** [comment]
I mean change static api

**Comment by abishekvashok** [comment]
@nolik2000 the api is already there, k what are you trying to do, I mean are you modifying the styles or adding a new feature or something, I can help you out @nolik2000 :)

**Comment by ghost** [comment]
And what is the next step?

**Comment by abishekvashok** [comment]
Ping me on slack, I don't want to flood this PR so let's keep this nice and tidy, ping me on slack please and we will discuss. :)

**Pull request closed without merging**